### DISCUSSION TITLE
how to add a keybinding to open a flatpak app?

### DISCUSSION THREAD

**Discussion by Ramy-Ashraf** [prasanthrangan/hyprdots#1045] [Q&A] [answered]
how to add a keybinding to open a flatpak app?

**Comment by kRHYME7** [comment]
``` flatpak list ```

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/b20c901b-085d-4da2-8c46-4ce39801e3d5)

**Reply by Ramy-Ashraf** [reply]
![image](https://github.com/prasanthrangan/hyprdots/assets/85342902/e9818d41-b73a-4522-84fb-dac60b4e0523)
![image](https://github.com/prasanthrangan/hyprdots/assets/85342902/fcd0821e-8cae-4403-b405-c69d6b38f43d)

Unfortunately, it doesn't work

**Reply by Ramy-Ashraf** [reply]
Also tried flatpak run dev.vencord.Vesktop but didn't work

**Reply by kRHYME7** [reply]
what happens if you run it on the terminal?

**Reply by kRHYME7** [reply, marked as answer]
Mine works. 
what file did you edited ? 

Edit
```
$HOME/.config/hypr/keybindings.conf
```

add this 
```
bind = $mainMod, D, exec,  flatpak run dev.vencord.Vesktop # Open Vesktop
```

or just 

```
bind = $mainMod, D, exec, dev.vencord.Vesktop # Open Vesktop
```

**Reply by Ramy-Ashraf** [reply]
"flatpak run dev.vencord.Vesktop" opens vesktop correctly
but for some reason can't use it  keybinding

**Reply by Ramy-Ashraf** [reply]
Oh, thanks it works correctly now

**Comment by Ramy-Ashraf** [comment]
> Mine works. what file did you edited ?
> 
> Edit
> 
> ```
> $HOME/.config/hypr/keybindings.conf
> ```
> 
> add this
> 
> ```
> bind = $mainMod, D, exec,  flatpak run dev.vencord.Vesktop # Open Vesktop
> ```
> 
> or just
> 
> ```
> bind = $mainMod, D, exec, dev.vencord.Vesktop # Open Vesktop
> ```
